src/framework/visualization/generateVisualization.ts

```diff
@@ -0,0 +1,34 @@
+import path from 'path';
+import { getRootStorageDirectory, writeFileToPath } from '../../fileSystem';
+import { generateVizHTML } from './generateVizHTML';
+import { retrieveIntegrationData } from './retrieveIntegrationData';
+
+/**
+ * Generates visualization of Vertices and Edges using https://visjs.github.io/vis-network/docs/network/
+ */
+export async function generateVisualization(): Promise<string> {
+  const integrationPath = getRootStorageDirectory();
+
+  const { entities, relationships } = await retrieveIntegrationData(
+    integrationPath,
+  );
+
+  const nodeDataSets = entities.map((entity) => ({
+    id: entity._key,
+    label: entity.name.toString(),
```

Comment:
I think `name` may be on most entities, but I believe will still use `displayName` in the graph viewer like you have with relationships below.

Reply:
Got it, updated it.